test/lib/cpd/CpdEngine.test.ts

```diff
@@ -52,22 +24,26 @@ describe('CpdEngine', () => {
 			const targets: RuleTarget[] = [{
 				target: '',
 				paths: ['file.cls','file.trigger','file.page','file.py','file.component']
-			}]
+			}];
 		
 			let langPathMap: Map<LANGUAGE, string[]> = new Map();
 			langPathMap.set(LANGUAGE.APEX, ['file.cls','file.trigger']);
 			langPathMap.set(LANGUAGE.VISUALFORCE, ['file.page','file.component']);
-			expect((testEngine as any).sortPaths(targets)).to.eql(langPathMap);
+			expect(testEngine.sortPaths(targets)).to.eql(langPathMap);
```

Comment:
What motivated the removal of the `testEngine as any` pattern?

Reply:
In general, we [avoid using `any` as much as possible](https://thoughtbot.com/blog/typescript-stop-using-any-there-s-a-type-for-that) in typescript, right. I think it was previously needed because `sortPath` method was private. I've moved it to default visibility and removed the `as any` pattern. I've also put a comment on why `sortPath` is not private. Since this was a one-off, I didn't go through the steps to make it protected and use a subclass.

Reply:
I agree that we should be careful about our use of `any`. But I think this is one of those times where it's worth using. Making a method public-scoped solely for the sake of testability feels wrong to me. Ideally, there'd be a `@TestVisible` annotation, but TS doesn't have that, so `as any` and subclassing are the only options.

Reply:
Agree, a `@TestVisible` annotation would've been perfect.
This method is only `default` visible and not `public` scoped. In fact, it has less visibility than `protected`, which would be needed for subclassing. It's visible here only because of the test module's location.
@jbartolotta-sfdc Any thoughts to help us weigh the options?

Reply:
@jfeingold35 Just read up. You're right. Converting this to subclassing.

Reply:
We had discussed the "as any" pattern for tests in this PR https://github.com/forcedotcom/sfdx-scanner/pull/310#discussion_r539673197 , Josh had converted a bunch of test code to use it. I think it is preferable for test code, it requires less code and allows methods to be private.